pages/about.html

```diff
@@ -0,0 +1,193 @@
+      With a dynamic line-up of keynote speakers, interactive panel discussions, and innovative
+      exhibitions, our online tech conference is the perfect platform to expand your knowledge,
+      grow your network, and unleash your potential in the exciting world of technology. Register
+      today and get ready to be inspired, challenged, and entertained!
+    </p>
+    <p id="intro-contact">
+      Please contact us via email for any further questions about ET Global Summit 2023!
+      <br >
+      <span>
+        <a href="mailto:etsummit2015@etusa.org">
+          etsummit2015@etusa.org
+        </a>
+        <span id="contact-caret-right">></span>
+      </span>
+    </p>
+  </section>
+
+
+  <section id="logo-section">
+    <h2 class="text-center">Emerging Technologies Global Summit 2023 Logo</h2>
+    <div class="line"></div>
+    <p class="text-center">
+      The logo of ET Global Summit 2023 was decided through the logo competition
+      from 9. January to 24. January.
+      <br>
+      <span id="extended-logo-competition-description">
+        After online voting and Selection Committee's judging process,
+        the logo from Abigail Trish won the prize.
+        <br>
+        After consultation with Ms. Trish, we optimized the prize-winning-logo
+        for this year's summit as following:
+      </span>
+    </p>
+    <div id="et-logo-div">
+      <img alt="et logo" src="../images/et_logo.PNG" />
+      <div>
+        <p class="logo-title m-0">
+          ET Global
+          <br >
+          Summit 2023
+        </p>
+        <p class="logo-competiton-date m-0 mt-1">
+          9th - 24th January 2023 | Maryland, USA
+        </p>
+      </div>
+    </div>
+  </section>
+
+
+  <section id="past-summits-section">
+    <h2>See the past ET Global Summits</h2>
+    <div class="line"></div>
+    <p>Take a look at the last three ET Global Summits which took place in Sweden, Tokyo, and in Berlin. </p>
+    <div id="past-summit-events">
+      <div class="summit-event">
+        <div>
+          <h3>2020</h3>
+          <p>ET Global Summit 2020 in Berlin</p>
+        </div>
+      </div>
+      <div class="summit-event">
+        <div>
+          <h3>2017</h3>
+          <p>ET Global Summit 2017 in Tokyo</p>
+        </div>
+      </div>
+      <div class="summit-event">
+        <div>
+          <h3>2014</h3>
+          <p>ET Global Summit 2014 in Sweden</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+
+  <section id="partners-section">
+    <h2>Partners</h2>
+    <div class="line"></div>
+    <div id="partners">
+      <img alt="" class="tesla-img" src="../images/tesla.png" />
+      <img alt="" class="google-img" src="../images/google.png" />
+      <img alt="" class="meta-img blur-image" src="../images/meta.png" />
+      <img alt="" class="microsoft-img blur-image" src="../images/microsoft.png" />
+      <img alt="" class="amazon-img blur-image" src="../images/amazon.png" />
```

Comment:
- [ ] While most of your images have an `alt` attribute, kindly don't forget to add it to the rest :+1: to make your page even more accessible.